src/Umbraco.Web/UmbracoHelper.cs

```diff
@@ -677,6 +677,19 @@ public IEnumerable<IPublishedContent> TypedContent(params object[] ids)
             return TypedContentForObjects(ids);
         }
 
+
+
+        /// <summary>
+        /// Gets the medias corresponding to the identifiers.

```

Comment:
As the method is `TypedContent`, could you update the xmldoc to use "content" instead of "medias"?